lib/reporters.go

```diff
@@ -59,6 +59,7 @@ func NewTextReporter(m *Metrics) Reporter {
 		"Latencies\t[mean, 50, 95, 99, max]\t%s, %s, %s, %s, %s\n" +
 		"Bytes In\t[total, mean]\t%d, %.2f\n" +
 		"Bytes Out\t[total, mean]\t%d, %.2f\n" +
+		"Content-Length\t[total, mean]\t%d, %.2f\n" +
```

Comment:
It may be confusing to users to read this in this report together with the other byte metrics. What do you think about calling this `BodySize` instead? Do you think it'd make it more self-documenting.

Reply:
I agree I think `BodySize` would make it more clear and self-documenting.  If someone was curious how this number was found they can look at the code to see it came from content-length and body size is more descriptive.

Reply:
I just realized that BodySize or ContentLength could refer to outgoing body or incoming.

So we should disambiguate that name to something like `ResponseBodySize` (or if you think of a better name, please suggest).

Reply:
does `ResponseBodyLength` gives more of a clue that it comes from `content-length` ?

Reply:
Works for me.

Reply:
this is done